VK/VK/Controllers/FriendsTableViewController.swift

```diff
@@ -99,4 +95,45 @@ final class FriendsTableViewController: UITableViewController {
         }
         sectionTitles = Array(sortedSectionsFriendMap.keys).sorted()
     }
+
+    private func fetchFriends() {
+        networkService.fetchFriends { [weak self] friend in
+            guard let self = self else { return }
+            switch friend {
+            case let .success(data):
+                self.realmService.saveData(data)
+                self.configureListFriends()
+                self.tableView.reloadData()
+            case let .failure(error):
+                print(error)
```

Comment:
```suggestion
                print(error.localizedDescription)
```
и по возможности алерт